### PR TITLE
use the first line of the description as the title

### PR DESCRIPTION
**Pull request by mengwong**
given a multiline description, use the first line as the title.

to get this behavior, run with `--desctitles`

Please:
- [X] Make your pull request atomic, fixing one issue at a time unless there are many relevant issues that cannot be decoupled.
- [X] Provide a test case & update the documentation in the `Readme.md`

### CODE REVIEW

File: typescript-json-schema.ts [hunk 1]
```diff
@@ -356,7 +358,17 @@ export class JsonSchemaGenerator {
         const comments = symbol.getDocumentationComment(this.tc);
 
         if (comments.length) {
-            definition.description = comments.map(comment => comment.kind === "lineBreak" ? comment.text : comment.text.trim().replace(/\r\n/g, "\n")).join("");
+            let description = comments.map(comment => comment.kind === "lineBreak" ? comment.text : comment.text.trim().replace(/\r\n/g, "\n")).join("\n").split(/\n/)
+
+            if (this.args.desctitles) {
+                let head = description.splice(0, 1)[0];
```
Comment (domoritz) [inline typescript-json-schema.ts]: You can just use pop, no?

File: typescript-json-schema.ts [hunk 2]
```diff
@@ -356,7 +358,17 @@ export class JsonSchemaGenerator {
...
+                definition.title = head;
+              // console.error(`setting title to the first line of description: ${definition.title}`)
```
Comment (domoritz) [inline typescript-json-schema.ts]: remove this

**Comment by stat92** [comment]
@mengwong It would be nice if you finish this PR.

**Comment by domoritz** [comment]
Please open a new pull request.

**Pull request closed without merging**